graylog2-server/src/main/java/org/graylog/plugins/sidecar/migrations/V20180212165000_AddDefaultCollectors.java

```diff
@@ -104,13 +104,154 @@ public void upgrade() {
                         "   - name: Security"
         );
         ensureCollector(
-                "nxlog",
+                "nxlog-linux",
```

Comment:
Unfortunately this resolves UI-wise to `nxlog-linux on Linux` when you select the collector on the configuration page.